packages/app/components/Nav/LeftNavBar.tsx

```diff
@@ -18,23 +19,33 @@ const MenuItem = styled(Menu.Item)`
   }
 `;
 
-const MenuItemLink = styled.a`
+const MenuItemLink = styled(Link)`
   padding: 10px 15px;
 `;
 
+interface LeftNavBarProps {
+  courseId: number;
+  queueId: number;
+}
 
-export default function LeftNavBar() {
+export default function LeftNavBar({ courseId, queueId }: LeftNavBarProps) {
   return (
     <HorizontalMenu mode="horizontal">
       <MenuItem key="today">
-        <MenuItemLink href="/today">Today</MenuItemLink>
+        <MenuItemLink href={`/class/${courseId}/today`}>Today</MenuItemLink>
```

Comment:
Good catch on using next/link, but we need to use the `as` prop or it won't be able to do link preloading.
https://nextjs.org/docs/api-reference/next/link#dynamic-routes
note: "href is a file system path used by the page and it shouldn't change at runtime"

Reply:
ohh awesome ok!